<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingFwVer</class>
 <widget class="QWidget" name="SettingFwVer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>save and  exit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_row">
      <property name="text">
       <string>Row:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_row"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_row">
      <property name="text">
       <string>select...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_lte">
      <property name="text">
       <string>LTE:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_lte"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_lte">
      <property name="text">
       <string>select...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>160</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lbl_prc">
      <property name="text">
       <string>PRC:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_prc"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_prc">
      <property name="text">
       <string>select...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
